src/containers/searchSection/SearchSection.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import Boards from '../../components/boards/Boards';
+import "./SearchSection.css";
+
+function SearchSection () {
+
+    const [selectedItem, setSelectedItem] = useState([])
+
+    // const [selectedCategory, setSelectedCategory] = useState([])
+    console.log(selectedItem)
```

Comment:
Line 9: if not going to be used, please delete it
Line 10: console.log's should be deleted,